tests/lib/prepare-restore.sh

```diff
@@ -344,11 +329,19 @@ EOF
     case "$SPREAD_SYSTEM" in
         debian-*|ubuntu-*)
             # in 16.04: apt build-dep -y ./
+            if [[ "$SPREAD_SYSTEM" == debian-* ]]; then
```

Comment:
Lets make this specific to   `debian-9-* ` - once we merge the debian-sid packaging we need to make sure not to mess around with debian/control to avoid hiding bugs in our packaging.